cmake/FindJansson.cmake

```diff
@@ -0,0 +1,15 @@
+if (Jansson_INCLUDE_DIRS AND Jansson_LIBRARY AND Jansson_VERSION)
+  set(Jansson_FOUND TRUE)
+else ()
+  find_path(Jansson_INCLUDE_DIRS NAMES jansson.h)
+  find_library(Jansson_LIBRARY NAMES jansson)
+
+  if (Jansson_INCLUDE_DIRS AND Jansson_LIBRARY)
+      set(regex_jansson_version "#define[ \t]+JANSSON_VERSION[ \t]+[\"]([^\"]+)[\"]")
+      file(STRINGS "${Jansson_INCLUDE_DIRS}/jansson.h" jansson_version REGEX "${regex_jansson_version}")
+      string(REGEX REPLACE "${regex_jansson_version}" "\\1" Jansson_VERSION "${jansson_version}")
+      unset(regex_jansson_version)
+      unset(jansson_version)
```

Comment:
Can we reuse the same variable here Jansson_VERSION and avoid unnecessary unsetting?

Reply:
All right, I changed it to use only one variable the final `Jansson_VERSION`.